### PR TITLE
Py3k builtins removal

### PR DESCRIPTION
**Pull request by Andrey-Raspopov**
This pull request addresses issue #2420 
There is an issue with open in bgzf that I'm aware of, dealing with it at the moment.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by Andrey-Raspopov** [comment]
I've found the mistake, we need to import open as _open in bgzf. Is it possible to rename bgzf "open" method to something else, to avoid builtin duplication, or is it too user-exposed?

**Comment by peterjc** [comment]
The use of ``Bio.bgzf.open`` is to match ``gzip.open`` etc - the docstrings describe the dangers of ``from Bio.bgzf import *`` due to the name collision and highlight ``from builtins import open`` as a workaround.

File: Bio/motifs/__init__.py [hunk 1]
```diff
@@ -179,7 +176,8 @@ def __init__(self, instances=None, alphabet=None):
             if self.length is None:
                 self.length = len(instance)
             elif self.length != len(instance):
-                message = "All instances should have the same length (%d found, %d expected)" % (len(instance), self.length)
+                message = "All instances should have the same length (%d found, %d expected)" % (
+                len(instance), self.length)
```
Comment (peterjc) [inline Bio/motifs/__init__.py]: Can you revert this unrelated change (at some point we will be apply black style to this file anyway)

File: Bio/motifs/__init__.py [hunk 2]
```diff
@@ -292,7 +290,8 @@ def __set_mask(self, mask):
         elif mask is None:
             self.__mask = (1,) * self.length
         elif len(mask) != self.length:
-            raise ValueError("The length (%d) of the mask is inconsistent with the length (%d) of the motif", (len(mask), self.length))
+            raise ValueError("The length (%d) of the mask is inconsistent with the length (%d) of the motif",
+                             (len(mask), self.length))
```
Comment (peterjc) [inline Bio/motifs/__init__.py]: Can you revert this unrelated change (at some point we will be apply black style to this file anyway)

File: NEWS.rst [hunk 3]
```diff
@@ -63,6 +63,7 @@ style has been reformatted with the ``black`` tool.
 Many thanks to the Biopython developers and community for making this release
 possible, especially the following contributors:
 
+- Andrey Raspopov
```
Comment (peterjc) [inline NEWS.rst]: I think this will be the first Biopython release you've contributed to, so you can add " (first contribution)" here too.

**Comment by peterjc** [comment]
Some minor edits aside, this looks good to me too. Thanks!

**Comment by peterjc** [comment]
Thank you for your patience, applied.
